### PR TITLE
Image refresh for rhel-7-5 - superseded by #8332

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for rhel-7-5
 * [x] image-refresh rhel-7-5

### CODE REVIEW

**Comment by martinpitt** [comment]
@mvollmer: Same issue as before, some VDO regression. Can you please have a look?

**Comment by mvollmer** [comment]
@martinpitt, screenshots are missing in the results directory, no?

**Comment by mvollmer** [comment]
> Same issue as before, some VDO regression. Can you please have a look?

Should be fixed.  The vdo command syntax got stricter.

**Comment by martinpitt** [comment]
@mvollmer: Thanks! One more? https://fedorapeople.org/groups/cockpit/logs/pull-8289-20171220-101515-6576709a-verify-rhel-7-5/log.html#100

Also, great to see udisks2-{lvm2,scsi} appearing at last, I'll send a PR to re-add the dependencies.

**Comment by mvollmer** [comment]
> One more?

Yes, the test assumed that there is no lvm2 support, so it has to be adapted.

**Comment by mvollmer** [comment]
@martinpitt, I have snuck in another fix.  Can you confirm the assumption that rewriting the condition in JavaScript actually makes it execute "atomically" with respect to the DOM?

File: test/verify/check-storage-partitions [hunk 1]
```diff
@@ -42,7 +42,8 @@ class TestStorage(StorageCase):
 
         dev = m.execute("dd if=/dev/zero of=/disk1 bs=1M count=10 && losetup --show loop12 /disk1").strip()
 
-        b.wait(lambda: b.is_present("#others") and dev in b.text("#others"))
+        b.wait_js_func('function (dev) { return ph_is_present("#others") && ph_text("#others").indexOf(dev) >= 0; }',
+                       dev)
```
Comment (martinpitt) [inline test/verify/check-storage-partitions]: The function wrapper is unnecessary - `b.wait_js_cond('ph_is_present("#others") && ph_text("#others").indexOf(dev) >= 0') is a bit simpler. But that's just FYI, I'm not vetoing this.

Strictly speaking this is not any more or less "atomic" than the previous variant. However, this has much less overhead as you don't need to go between Python, Chrome, and JS for each iteration, so it's a nice improvement for sure.

**Comment by martinpitt** [comment]
@mvollmer: Is rhel-7-5 the only image that has `vdo`? If not, the tests on the other OSes need triggering too

**Comment by mvollmer** [comment]
> Is rhel-7-5 the only image that has vdo

Yes.

**Comment by mvollmer** [comment]
> I have snuck in another fix.

I shouldn't have done this.  I make a separate PR for it.

**Comment by mvollmer** [comment]
> I shouldn't have done this. I make a separate PR for it.

#8309

**Comment by martinpitt** [comment]
The image timed out over the holidays, so this became stale. I'm building a new rhel-7-5 image in #8332, and I copied @mvollmer's commits there.

Keeping this open to prevent the bots from opening yet another one. This can be closed once #8332 lands.

**Pull request closed without merging**